pkg/webhook/cspc_operations.go

```diff
@@ -509,3 +470,203 @@ func GetNewBDFromRaidGroups(newRG, oldRG *cstor.RaidGroup) map[string]string {
 	newToOldBlockDeviceMap[newBD] = oldBD
 	return newToOldBlockDeviceMap
 }
+
+// raidGroups contains list of oldraid groups and newraid groups
+type raidGroups struct {
+	oldRaidGroups []cstor.RaidGroup
+	newRaidGroups []cstor.RaidGroup
+	rgType        string
+}
+
+func getNewBDsFromStripeSpec(oldRg, newRg cstor.RaidGroup) []string {
+	mapOldRg := map[string]bool{}
```

Comment:
nit: good to change the variable name to mapOldBlockDevices so that it will be meaningful